test/test_file_utils.py

```diff
@@ -212,3 +213,194 @@ def test_guess_project_dir(tmp_path: Path) -> None:
     with cwd(str(tmp_path)):
         result = guess_project_dir(None)
         assert result == str(tmp_path)
+
+
+BASIC_PLAYBOOK = """
+- name: "playbook"
+  tasks:
+    - name: hello
+      debug:
+        msg: 'world'
+"""
+
+
+@pytest.fixture
+def tmp_updated_lintable(
+    tmp_path: Path, path: str, content: str, updated_content: str
+) -> Lintable:
+    """Create a temp file Lintable with a content update that is not on disk."""
+    lintable = Lintable(tmp_path / path, content)
+    with lintable.path.open("w", encoding="utf-8") as f:
+        f.write(content)
```

Comment:
Why didn't you just do 
```suggestion
    lintable.path.write_text(content)
```
?

Reply:
Reasoning is in https://github.com/ansible-community/ansible-lint/pull/1884#discussion_r807134835